common/build/build-common/api-extractor-lint.entrypoint.json

```diff
@@ -20,5 +20,17 @@
 	"tsdocMetadata": {
 		// Don't generate docs metadata in lint pass
 		"enabled": false
+	},
+	"messages": {
+		"extractorMessageReporting": {
+			// Since we generate API reports for different release levels, it is possible (and valid) for `@link` tags
+			// to reference an API item that won't be visible at that level.
+			// For example, a public API may note that a newer feature is available in a beta release and link to that item.
+			// We want that to be possible, so we suppress this error for report generation.
+			// It will be enabled for `model` generation, which is used for generating API documentation for the entire suite.
```

Comment:
```suggestion
			// Since we lint entrypoints at different release levels, it is possible (and valid) for `@link` tags
			// to reference an API item that won't be visible at that level.
			// For example, a public API may note that a newer feature is available in a beta release and link to that item.
			// We want that to be possible, so we suppress this error for report generation.
			// It will be enabled for `model` generation, which is used for generating API documentation for the entire suite.
```